data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -231,10 +231,12 @@ def _create_tile_data(
         score_tiles = score_tiles[
             ~score_tiles[field_names.GEOID_TRACT_FIELD].isin(tracts_to_drop)
         ]
-
-        score_tiles[constants.TILES_SCORE_FLOAT_COLUMNS] = score_tiles[
-            constants.TILES_SCORE_FLOAT_COLUMNS
-        ].apply(
+        float_cols = [
+            col
+            for col, col_dtype in score_tiles.dtypes.items()
+            if col_dtype == np.dtype("float64")
+        ]
+        score_tiles[float_cols] = score_tiles[float_cols].apply(
```

Comment:
i think since we are FLOORING everything to two decimal places we no longer need to floor it in this way but this is correct otherwise. 

A question -- why use `==`vs  `is` for the comparison (earnest question, not a gotcha)

Reply:
Honestly because I have a heuristic for using "is" on singletons like None, True, False and my heuristic doesn't extend to dtypes :)

What way would you floor things now that we're flooring everything to two decimal places?

Reply:
i don't understand here why we use a separate floor series rather than applying `floor`, a standard function?